ansible/roles/ocp-workload-aiedge-computing-blueprint/tasks/workload.yml

```diff
@@ -0,0 +1,578 @@
+
+# Expose internal registry for core cluster
+- name: Expose the OpenShift internal registry
+  k8s:
+    state: present
+    definition:
+      apiVersion: imageregistry.operator.openshift.io/v1
+      kind: Config
+      metadata:
+        name: cluster
+      spec:
+        defaultRoute: true
+
+- name: Get the OpenShift registry route object
+  k8s_info:
+    name: default-route
+    api_version: route.openshift.io/v1
+    kind: route
+    namespace: openshift-image-registry
+  register: image_registry_object
+
+- name: Set the url for the exposed image registry url
+  set_fact:
+    core_imageregistry_url: "{{ image_registry_object.resources[0].spec.host }}"
+
+- name: Get the gitea route
+  k8s_info:
+    name: gitea
+    api_version: route.openshift.io/v1
+    kind: route
+    namespace: gitea
+    label_selectors:
+      - "app = gitea"
+  register: gitea_route_object
+
+- name: Set the gitea server host
+  set_fact:
+    gitea_server_host: "{{ gitea_route_object.resources[0].spec.host }}"
+
+- name: Set the url gitea server and organization
+  set_fact:
+    gitea_server_host_org: "{{ gitea_server_host }}/{{ gitea_user }}"
+    gitea_repo_org_url: "https://{{ gitea_server_host }}/{{ gitea_user }}"
+  vars:
+    gitea_user: "{{ ocp_workload_aiedge_computing_blueprint.gitea_user }}"
+
+
+####################################################################################################
+# GIT REPO SETUP
+####################################################################################################
+- name: Set the temporary location directory where the repo will be saved
+  tempfile:
+    state: directory
+    suffix: -redhat-edge-computing
+  register: edge_computing_root_dir
+
+- set_fact:
+    hub_repo_dir: "{{ edge_computing_root_dir.path }}/blueprint-management-hub"
+    edge_repo_dir: "{{ edge_computing_root_dir.path }}/blueprint-industrial-edge"
+    manuela_dev_repo_dir: "{{ edge_computing_root_dir.path }}/manuela-dev"
+    manuela_gitops_repo_dir: "{{ edge_computing_root_dir.path }}/manuela-gitops"
+
+# Clone the manuela dev & gitops repo and create a branch for the demo
+- name: Clone the MANUela repos
+  git:
+    repo: "{{ item.repo }}"
+    dest: "{{ item.dest }}"
+    refspec: '+refs/tags/*:refs/tags/*'
+  loop:
+    # yamllint disable rule:line-length
+    - {repo: "{{ gitea_repo_org_url }}/{{ ocp_workload_aiedge_computing_blueprint.acm_blueprint_hub_repo_name }}",
+       dest: "{{ hub_repo_dir }}"}
+    - {repo: "{{ gitea_repo_org_url }}/{{ ocp_workload_aiedge_computing_blueprint.acm_blueprint_edge_repo_name }}",
+       dest: "{{ edge_repo_dir }}"}
+    - {repo: "{{ gitea_repo_org_url }}/{{ ocp_workload_aiedge_computing_blueprint.manuela_dev_repo_name }}",
+       dest: "{{ manuela_dev_repo_dir }}"}
+    - {repo: "{{ gitea_repo_org_url }}/{{ ocp_workload_aiedge_computing_blueprint.manuela_gitops_repo_name }}",
+       dest: "{{ manuela_gitops_repo_dir }}"}
```

Comment:
Please use normal YAML syntax rather than JSON syntax:
```
  loop:
    - repo: "{{ gitea_repo_org_url }}/{{ ocp_workload_aiedge_computing_blueprint.acm_blueprint_hub_repo_name }}
      dest: "{{ hub_repo_dir }}
```

Reply:
Updated. I can't remember why I chose to use the json brackets